.github/ci-scripts/bash/functions-for-build

```diff
@@ -565,6 +591,61 @@ setup_qa_env() {
   deactivate
 }
 
+setup_packer_env() {
+  sudo python3 -m pip uninstall -y packer_framework
+  python3 -m pip install --user virtualenv --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m venv pkenv
+  source pkenv/bin/activate
+  python3 -m pip install wheel --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m pip install -r ./requirements.txt --extra-index-url=https://$GITHUB_USER:$GITHUB_TOKEN@nexus.tessell.cloud/repository/${NEXUS_PULL_REPOS_PY}/simple
```

Comment:
Why `nexus.tessell.cloud`? Use the env variable that will point to local name of nexus.

Reply:
I was using that only previously, 
but at some point during testing, [this error started coming up](https://github.com/TessellDevelopment/ci-test-repo-b/actions/runs/11051948155/job/30703028625) 
I tried the same command from inside pod also and there also same issue was occurring,
(to unblock myself I made the above changes "as I thought it to be an intermediary issue")